src/components/tag/_tag.scss

```diff
@@ -0,0 +1,9 @@
+@import 'settings/census';
```

Comment:
We shouldn't need to do this. If the page template has the theme "census" the variables should be applied automatically. The way the theme styles work is to create a variable in the base `vars/colors` file and provide a default colour for that variable. The variable is then overwritten in the theme scss file.

Reply:
I will try this again. For some reason it was not working for me.

Reply:
This is now working. I added the colour into 'vars/colors' also as you suggested.